lib/crypto-algorithms/olm.js

```diff
@@ -185,4 +185,5 @@ OlmDecryption.prototype.decryptEvent = function(event) {
     }
 };
 
-base.registerAlgorithm(olmlib.OLM_ALGORITHM, OlmEncryption, OlmDecryption);
+base.registerAlgorithm(olmlib.OLM_V2_ALGORITHM, OlmEncryption, OlmDecryption);
```

Comment:
Why do you need a OLM_V2_ALGORITHM here? I don't think anything in the olm ratchet changed?